samples/sample_multi_transcode/include/pipeline_transcode.h

```diff
@@ -131,6 +139,10 @@ namespace TranscodingSample
     struct sMctfRunTimeParam
     {
 #ifdef ENABLE_MCTF_EXT
+#if 0
```

Comment:
Why do we have #if 0 here? We already have define right above.

Reply:
Will be removed.